data/formations.txt

```diff
@@ -0,0 +1,649 @@
+# Copyright (c) 2019 by Peter van der Meer
```

Comment:
Shouldn't you update the date?

Reply:
This file was indeed also changed in 2020.